components/SingleVcItem.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { useContext, useRef } from 'react';
```

Comment:
we have combined the implementation of singlevcItem and vcItem together, can we still merge two files into single one, or are we having any different implementation for both?

Reply:
We are using SingleVcitem.tsx in the sendVcScreen component when there was only one card in the wallet. Because it should select automatically.

Reply:
Removed unused file SingleVcItem.tsx from the app code.